packages/xo-web/src/xo-app/settings/remotes/index.js

```diff
@@ -99,7 +99,9 @@ const COLUMN_SPEED = {
     const benchmark = get(() => remote.benchmarks[remote.benchmarks.length - 1])
 
     return (
-      benchmark !== undefined && (
+      benchmark !== undefined &&
+      benchmark.readRate !== undefined &&
+      benchmark.writeRate !== undefined && (
```

Comment:
How can `benchmark` be defined but not `readRate` or `writeRate`?

Due to previous implementations?

Reply:
It should not happen if xo-server and fs are on the same page. So this is just an extra protection. What do you think?

Reply:
Good idea :+1: